app/src/main/java/com/infomaniak/mail/utils/MessageBodyUtils.kt

```diff
@@ -38,6 +38,8 @@ object MessageBodyUtils {
 
     private const val QUOTE_DETECTION_TIMEOUT = 1_500L
 
+    private const val MESSAGE_LENGTH_LIMIT = 104_448
```

Comment:
Maybe add a comment to explain the reasoning behind this strange value so we know why it's here and what we can do with it when we will have forgotten the choice we made